<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CVE-2014-4210 SSRF PoRt Sc4nNer by Unmanarc</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
    border-style: outset;
    border-width: 6px;
    border-color: red;</string>
      </property>
      <property name="text">
       <string>Scan 'em all!</string>
      </property>
      <property name="icon">
       <iconset resource="../rc.qrc">
        <normaloff>:/icons/iconfinder_Hacker_379509.svg</normaloff>:/icons/iconfinder_Hacker_379509.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Remote Host/IP to Scan:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>From TCP Port:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Intermediate SSRF vulnerable URL:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_scanhost">
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="spinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit_url">
      <property name="text">
       <string>https://testsite.com/uddiexplorer/SearchPublicRegistries.jsp</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../rc.qrc"/>
 </resources>
 <connections/>
</ui>
